scripts/beautify.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Reformat C++ code using clang-format
+
+# note: with the -style=file option, clang-format reads the config from ./.clang-format
+# See here for the full list of supported options: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
+FORMAT_CMD="clang-format -i -style=file"
+
+# Filter out any files that shouldn't be auto-formatted.
+# note: -v flag inverts selection - this tells grep to *filter out* anything
+#       that matches the pattern. For testing, you can remove the -v to ssee
+#       which files would have been excluded.
+FILTER_CMD="grep -v -E objects|src/polyclipping|src/CGAL_Nef3_workaround.h|src/CGAL_workaround_Mark_bounded_volumes.h|src/convex_hull_3_bugfix.h|src/OGL_helper.h"
+
+function reformat_all() {
+    find . -name "*.h" -o -name "*.hpp" -o -name "*.cc" -o -name "*.cpp" \
```

Comment:
Perhaps limit the find to `src` for now to avoid it picking up too much other stuff